flow-typed/Result.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+type Result<Error, Value> =
+  | { tag: 'Ok', value: Value }
+  | { tag: 'Error', error: Error };
```

Comment:
Could we add a comment explaining this? Is is a work around for the prettier issue?

Reply:
Thanks!